airflow/providers/cncf/kubernetes/provider.yaml

```diff
@@ -97,13 +97,13 @@ dependencies:
   # limiting minimum airflow version supported in cncf.kubernetes provider, due to the
   # potential breaking changes in Airflow Core as well (kubernetes is added as extra, so Airflow
   # core is not hard-limited via install-requirements, only by extra).
-  - kubernetes>=28.1.0,<=29.0.0
```

Comment:
Can we so that?
According to
https://github.com/kubernetes-client/python?tab=readme-ov-file#compatibility-matrix-of-supported-client-versions

> client 29.y.z: Kubernetes 1.28 or below (+-), Kubernetes 1.29 (✓), Kubernetes 1.30 or above (+-)

While we promise support for 1.26, 1.27, 1.28, 1.29